### PR TITLE
add documentation for zwave_js.set_config_parameter service

### PR DESCRIPTION
**Pull request by raman325**
## Proposed change
Add documentation for new service

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/46673
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by cgarwood — CHANGES_REQUESTED** [review]
We need to remove `- Support for setting configuration parameters through service calls is currently not supported but may be added in a later release.` from the Current Limitations section

File: source/_integrations/zwave_js.markdown [hunk 1]
```diff
@@ -55,6 +55,51 @@ Advanced users: Make sure that the server started successfully by inspecting the
 
 ## Services
 
+### Service `zwave_js.set_config_parameter`
+
+This service will update a configuration parameter. At this time, it is not possible to update multiple partial parameters in a single call, but we hope to add support for that in the future.
+
+| Service Data Attribute 	| Required  	| Description                                                                                                                               	|
+|------------------------	|-----------	|-------------------------------------------------------------------------------------------------------------------------------------------	|
+| `entity_id`            	| Exclusive 	| Entity (or list of entities) to set the configuration parameter on. Either this or `device_id` must be provided.                       	|
```
Comment (MartinHjelmare) [inline source/_integrations/zwave_js.markdown]: We changed it and entity_id and device_id are no longer exclusive.

Comment (raman325) [reply source/_integrations/zwave_js.markdown]: thanks for catching that

**Review by MartinHjelmare — DISMISSED** [review]

File: source/_integrations/zwave_js.markdown [hunk 2]
```diff
@@ -55,6 +55,51 @@ Advanced users: Make sure that the server started successfully by inspecting the
...
+Let's use parameter 31 for [this device](https://devices.zwave-js.io/?jumpTo=0x000c:0x0203:0x0001:0.0) as an example to show examples of different ways that the `LED 1 Blink Status (bottom)` partial parameter can be set. Note that in places where we are using different values for the same key, the different values are interchangeable across the examples. We can, for instance, use `1` or `Blink` interchangeably for the `value` in all of the examples.
+
+Example 1:
+```yaml
+service: zwave_js.set_config_parameter
+data:
+  entity_id: switch.fan
+  parameter: 31
+  bitmask: 0x01
+  value: 1
```
Comment (frenck) [inline source/_integrations/zwave_js.markdown]:
```suggestion
service: zwave_js.set_config_parameter
target:
  entity_id: switch.fan
data:
  parameter: 31
  bitmask: 0x01
  value: 1
```

File: source/_integrations/zwave_js.markdown [hunk 3]
```diff
@@ -55,6 +55,51 @@ Advanced users: Make sure that the server started successfully by inspecting the
...
+```
+
+Example 2:
+```yaml
+service: zwave_js.set_config_parameter
+data:
+  entity_id: switch.fan
+  parameter: 31
+  bitmask: 1
+  value: Blink
```
Comment (frenck) [inline source/_integrations/zwave_js.markdown]:
```suggestion
service: zwave_js.set_config_parameter
target:
  entity_id: switch.fan
data:
  parameter: 31
  bitmask: 1
  value: "Blink"
```

File: source/_integrations/zwave_js.markdown [hunk 4]
```diff
@@ -55,6 +55,51 @@ Advanced users: Make sure that the server started successfully by inspecting the
...
+```
+
+Example 3:
+```yaml
+service: zwave_js.set_config_parameter
+data:
+  entity_id: switch.fan
+  parameter: "LED 1 Blink Status (bottom)"
+  value: Blink
```
Comment (frenck) [inline source/_integrations/zwave_js.markdown]:
```suggestion
service: zwave_js.set_config_parameter
target:
  entity_id: switch.fan	
data:
  entity_id: switch.fan
  parameter: "LED 1 Blink Status (bottom)"
  value: "Blink"
```

**Review by frenck — APPROVED** [review]

**Pull request merged**